ruby/lib/rspec/queue.rb

```diff
@@ -174,6 +174,12 @@ def run_specs(example_groups)
 
       private
 
+      def shuffle(examples)
+        return tests unless RSpec::Queue.config.seed
```

Comment:
Should you return `examples` rather than `tests` here?

Reply:
🤦‍♂️